aptos-move/framework/src/natives/aggregator_natives/aggregator_v2.rs

```diff
@@ -62,58 +62,17 @@ pub const EAGGREGATOR_FUNCTION_NOT_YET_SUPPORTED: u64 = 0x03_0009;
 /// If we want to increase this, we need to modify BITS_FOR_SIZE in types/src/delayed_fields.rs.
 pub const DERIVED_STRING_INPUT_MAX_LENGTH: usize = 1024;
 
-/// Given the native function argument and a type, returns a tuple of its
-/// fields: (`aggregator id`, `max_value`).
-pub fn get_aggregator_fields_by_type(
-    ty_arg: &Type,
-    agg: &StructRef,
-) -> SafeNativeResult<(u128, u128)> {
-    match ty_arg {
-        Type::U128 => {
-            let id = get_struct_field(agg, AGG_VALUE_FIELD_INDEX)?.value_as::<u128>()?;
-            let max_value = get_struct_field(agg, AGG_MAX_VALUE_FIELD_INDEX)?.value_as::<u128>()?;
-            Ok((id, max_value))
-        },
-        Type::U64 => {
-            let id = get_struct_field(agg, AGG_VALUE_FIELD_INDEX)?.value_as::<u64>()?;
-            let max_value = get_struct_field(agg, AGG_MAX_VALUE_FIELD_INDEX)?.value_as::<u64>()?;
-            Ok((id as u128, max_value as u128))
-        },
-        _ => Err(SafeNativeError::Abort {
-            abort_code: EUNSUPPORTED_AGGREGATOR_TYPE,
-        }),
-    }
-}
-
-pub fn get_snapshot_field_by_type(
-    ty_arg: &Type,
-    agg_snapshot: &StructRef,
-) -> SafeNativeResult<u128> {
-    match ty_arg {
-        Type::U128 => {
-            let value = get_struct_field(agg_snapshot, AGG_SNAPSHOT_VALUE_FIELD_INDEX)?
-                .value_as::<u128>()?;
-            Ok(value)
-        },
-        Type::U64 => {
-            let value = get_struct_field(agg_snapshot, AGG_SNAPSHOT_VALUE_FIELD_INDEX)?
-                .value_as::<u64>()?;
-            Ok(value as u128)
-        },
-        _ => Err(SafeNativeError::Abort {
-            abort_code: EUNSUPPORTED_AGGREGATOR_SNAPSHOT_TYPE,
-        }),
-    }
-}
-
-pub fn get_derived_string_field(derived_string: &StructRef) -> SafeNativeResult<Vec<u8>> {
-    Ok(string_to_bytes(
-        get_struct_field(derived_string, DERIVED_STRING_VALUE_FIELD_INDEX)?.value_as::<Struct>()?,
-    )
-    .map_err(PartialVMError::from)?)
+macro_rules! abort_if_not_enabled {
```

Comment:
abort_if_aggregator_api_not_enabled?
abort_if_no_aggregator_api?
otherwise it's too generic, it is private so no big deal, but still

Reply:
done